app/views/projects/shared/_users_table.html.erb

```diff
@@ -15,7 +15,7 @@
       <td class="govuk-table__cell"><%= user.email %></td>
       <td class="govuk-table__cell"><%= user.conversion_count %></td>
       <td class="govuk-table__cell">
-        <%= link_to t("project.table.body.view_projects_html", trust_name: user.name), by_user_team_projects_path(user.id) %>
+        <%= link_to t("project.table.body.view_projects_html", trust_name: user.name), by_user_team_projects_path(user.id) if user.conversion_count > 0 %>
```

Comment:
Would you mind switching this locale to the `view_projects_for_html` which accepts 
'entity' rather than 'trust'?